src/odemis/driver/hamamatsurx.py

```diff
@@ -877,7 +877,7 @@ class StreakCamera(model.HwComponent):
     Client to connect to HPD-TA software via RemoteEx.
     """
 
-    def __init__(self, name, role, port, host, children=None, daemon=None, **kwargs):
```

Comment:
All the changes done on hamamatsurx should be done identically to simstreakcam.py.